include/deal.II/base/geometry_info.h

```diff
@@ -813,7 +813,8 @@ class RefinementCase : public RefinementPossibilities<dim>
    * mapping from the symbolic flags defined in the RefinementPossibilities
    * base class to actual numerical values (the array indices).
    */
-  DEAL_II_HOST_DEVICE operator std::uint8_t() const;
+  DEAL_II_HOST_DEVICE
+  operator std::uint8_t() const;
```

Comment:
I think you will want to undo this change and the other one in this file -- deal.II is indented automatically by a script, and github will complain if you have indentations that do not match what the script wants to do. Just go back to how it was originally.